### ISSUE TITLE
Needed --lat and --lng options

### ISSUE THREAD

**Issue by zzolo** [veltman/address-book#4]
Here's the original data:

```
address
0 JFK AIRPORT JAMAICA 11430
0 jfk airport queens 11430
0 JFK INT'L AIRPORT JAMAICA 11430
0 JFK INTERNATIONAL AIRPOR JAMAICA 11430
0 JFK INTERNATIONAL AIRPORT JAMAICA 11430
0 JFK INTL AIRPORT JAMAICA 11430
```

Running with: `csvgeocode addresses.csv addressesGeo.csv --url="https://maps.googleapis.com/maps/api/geocode/json?address={{address}}" --handler=google --verbose`

```
address,undefined
0 JFK AIRPORT JAMAICA 11430,-74.8987696
0 jfk airport queens 11430,-74.8987696
0 JFK INT'L AIRPORT JAMAICA 11430,-74.8987696
```

Running with: `csvgeocode addresses.csv addressesGeo.csv --url="https://maps.googleapis.com/maps/api/geocode/json?address={{address}}" --handler=google --verbose --lat=lat --lng=lng`

```
address,lat,lng
0 JFK AIRPORT JAMAICA 11430,40.123,-74.8987696
0 jfk airport queens 11430,40.123,-74.8987696
0 JFK INT'L AIRPORT JAMAICA 11430,40.123,-74.8987696
```

Might be something to do with the single column.

(side note, I got an error without the explicit `--handler=google` option)

**Comment by veltman** [comment]
Hmm, I don't experience this problem at all.  Are you running v2.0.1?  I also don't get an error if I omit `--handler google` (or `--handler=google`)

**Comment by zzolo** [comment]
Using 2.0.2.  Just did `npm install -g csvgeocode` today.

So, if I checkout master, do `npm install`, and run the following, I get an error:

```
./bin/csvgeocode ~/Downloads/addresses.csv --url "https://maps.googleapis.com/maps/api/geocode/json?address={{address}}"
/Users/zzolo/Code/forks/csvgeocode/src/csvgeocode.js:43
    throw new TypeError("Invalid value for 'handler' option.  Must be the name
          ^
TypeError: Invalid value for 'handler' option.  Must be the name of a built-in handler or a custom handler.
    at generate (/Users/zzolo/Code/forks/csvgeocode/src/csvgeocode.js:43:11)
    at Object.<anonymous> (/Users/zzolo/Code/forks/csvgeocode/bin/csvgeocode:48:14)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

Though, if I add `--handler google` it works.

Then if I run the following:

```
./bin/csvgeocode ~/Downloads/addresses.csv ~/Downloads/addresses-geocoded.csv --url "https://maps.googleapis.com/maps/api/geocode/json?address={{address}}" --handler google
```

It produces the following file (without error):

```
address,undefined
0 JFK AIRPORT JAMAICA 11430,-73.7821139
0 jfk airport queens 11430,-73.7821139
0 JFK INT'L AIRPORT JAMAICA 11430,-73.7846488
0 JFK INTERNATIONAL AIRPOR JAMAICA 11430,-73.7821139
0 JFK INTERNATIONAL AIRPORT JAMAICA 11430,-73.7821139
0 JFK INTL AIRPORT JAMAICA 11430,-73.7821139
0 LA GUARDIA AIRPORT FLUSHING 11369,-73.8619485
0 LAGUARDIA AIRPORT FLUSHING 11369,-73.8619485
```

And if I add an extra column, I still get the same issue.

If I add the `lat` and `lng` arguments, it works.

```
./bin/csvgeocode ~/Downloads/addresses.csv ~/Downloads/addresses-geocoded.csv --url "https://maps.googleapis.com/maps/api/geocode/json?address={{address}}" --handler google --lat lat --lng lng
```

```
address,lat,lng
0 JFK AIRPORT JAMAICA 11430,40.6435529,-73.7821139
0 jfk airport queens 11430,40.6435529,-73.7821139
0 JFK INT'L AIRPORT JAMAICA 11430,40.6484307,-73.7846488
0 JFK INTERNATIONAL AIRPOR JAMAICA 11430,40.6435529,-73.7821139
0 JFK INTERNATIONAL AIRPORT JAMAICA 11430,40.6435529,-73.7821139
0 JFK INTL AIRPORT JAMAICA 11430,40.6435529,-73.7821139
0 LA GUARDIA AIRPORT FLUSHING 11369,40.7690046,-73.8619485
0 LAGUARDIA AIRPORT FLUSHING 11369,40.7690046,-73.8619485
0 PENN STATION NEW YORK 10001,40.750568,-73.993519
```

**Comment by veltman** [comment]
This was a problem with undefined stuff in `yargs`, fixed in 6bc185bc
